hbase-server/src/main/java/org/apache/hadoop/hbase/wal/AbstractWALRoller.java

```diff
@@ -255,22 +248,29 @@ public void close() {
     private final WAL wal;
     private final AtomicBoolean rollRequest;
     private long lastRollTime;
+    private boolean isRolling;
```

Comment:
I think this field will be accessed by multiple threads so a simple boolean it not enough to keep us safe?